netlify-plugin-cloudinary/src/index.ts

```diff
@@ -241,26 +241,38 @@ export async function onBuild({
           const cldAssetPath = `/${path.join(PUBLIC_ASSET_PATH, mediaPath)}`;
           const cldAssetUrl = `${host}${cldAssetPath}`;
 
-          const { cloudinaryUrl: assetRedirectUrl } = await getCloudinaryUrl({
-            deliveryType: 'fetch',
-            folder,
-            path: `${cldAssetUrl}/:splat`,
-            uploadPreset,
-          });
-
-          netlifyConfig.redirects.unshift({
-            from: `${cldAssetPath}/*`,
-            to: `${mediaPath}/:splat`,
-            status: 200,
-            force: true,
-          });
-
-          netlifyConfig.redirects.unshift({
-            from: `${mediaPath}/*`,
-            to: assetRedirectUrl,
-            status: 302,
-            force: true,
-          });
+          try {
+
+            const { cloudinaryUrl: assetRedirectUrl } = await getCloudinaryUrl({
+              deliveryType: 'fetch',
+              folder,
+              path: `${cldAssetUrl}/:splat`,
+              uploadPreset,
+            });
+
+            netlifyConfig.redirects.unshift({
+              from: `${cldAssetPath}/*`,
+              to: `${mediaPath}/:splat`,
+              status: 200,
+              force: true,
+            });
+
+            netlifyConfig.redirects.unshift({
+              from: `${mediaPath}/*`,
+              to: assetRedirectUrl,
+              status: 302,
+              force: true,
+            });
+          } catch (error) {
+            console.error('Error', error);
```

Comment:
probably dont need this here given you're logging the errors from within the function itself, correct?